addons/website_profile/views/website_profile.xml

```diff
@@ -338,7 +338,7 @@
             </svg>
             <div class="o_pc_overlay d-flex flex-column align-items-center justify-content-center">
                 <img class="img-fluid"
-                    t-attf-src="/web/image/gamification.karma.rank/#{user.next_rank_id.id if user.next_rank_id else user.rank_id.id}/image_medium"
+                    t-attf-src="website.image_url(#{user.next_rank_id if user.next_rank_id else user.rank_id}, 'image_medium')"
```

Comment:
Nop, this should not be a `attf` anymore, it will display plain text `website.image_url(gamification.karma.rank(5,), 'image_medium')` in DOM src.

```xml
t-att-src="website.image_url(user.next_rank_id if user.next_rank_id else user.rank_id, 'image_medium')"
```
should be fine